settings.gradle

```diff
@@ -1,7 +1,22 @@
 plugins {
-  id "com.gradle.enterprise" version "3.4.1"
+    id "com.gradle.enterprise" version "3.9"
+    id 'com.gradle.common-custom-user-data-gradle-plugin' version '1.6.5'
 }
 
+gradleEnterprise {
+    server = "https://gradle.a8c.com"
+    allowUntrustedServer = true // Unless a trusted certificate is already configured in GE.
```

Comment:
We don't need this line, because we have a trusted SSL cert 🙂

Reply:
Great, thank you, I wasn't sure about it so I defaulted to trust the world 😅  will remove it! 🥇

Reply:
Done: 4169fc7a00451e4cdd36cec4610f31d79d88d1d7